src/main/resources/db/changelog/changes/v1.1.0/sql/entity-type-definitions/insert-organization-definition.sql

```diff
@@ -0,0 +1,294 @@
+                                                  "value": "inactive"
+                                             },
+                                             {
+                                                   "label": "Pending",
+                                                   "value": "pending"
+                                             }
+                                      ],
+                            "dataType": {
+                              "dataType": "stringType"
+                                   },
+                            "valueGetter": "org.jsonb->>''status''",
+                            "filterValueGetter": "${tenant_id}_mod_organizations_storage.f_unaccent(org.jsonb ->> ''status''::text)",
+                            "valueFunction": "${tenant_id}_mod_organizations_storage.f_unaccent(:value)",
+                            "visibleByDefault": false
+                 },
+                 {
+                            "name": "description",
+                            "dataType": {
+                              "dataType": "stringType"
+                                   },
+                            "valueGetter": "org.jsonb->>''description''",
+                            "visibleByDefault": false
+                 },
+                {
+                            "name": "accounting_code",
+                            "dataType": {
+                              "dataType": "stringType"
+                                   },
+                            "filterValueGetter": "${tenant_id}_mod_organizations_storage.f_unaccent(org.jsonb ->> ''erpCode''::text)",
+                            "valueFunction": "${tenant_id}_mod_organizations_storage.f_unaccent(:value)",
+                            "valueGetter": "org.jsonb->>''erpCode''",
+                            "visibleByDefault": false
+                 },
+                 {
+                            "name": "payment_method",
+                            "values": [
+                                             {
+                                                   "label": "Credit Card",
+                                                   "value": "Credit Card"
+                                              },
+                                              {
+                                                   "label": "Cash",
+                                                   "value": "Cash"
+                                              },
+                                              {
+                                                    "label": "Physical Check",
+                                                    "value": "Physical Check"
+                                              },
+                                              {
+                                                   "label": "EFT",
+                                                   "value": "EFT"
```

Comment:
Sanity check: Should this be EFT or ETF? I've never heard of EFT, but ETF wouldn't really make a lot of sense either, so I just wanna be sure :P

Reply:
Electronic Funds Transfer

Reply:
@mweaver-ebsco , I double checked with database its ETF only :P